plugins/org.python.pydev.ast/src/org/python/pydev/ast/codecompletion/revisited/modules/SourceModule.java

```diff
@@ -694,6 +696,14 @@ public TokensList getCompletionsForBase(ICompletionState initialState, ICodeComp
             state = initialState.getCopy();
             state.setActivationToken(s);
             return manager.getCompletionsForModule(this, state);
+        } else if (classDef.bases[baseIndex] instanceof Subscript) {
+            TypeInfo type = new TypeInfo(classDef.bases[baseIndex]);
+            if ("ContextManager".equals(type.getActTok())) {
```

Comment:
I think that you don't need to special case `"ContextManager"` here... (this may also be the fix for https://www.brainwy.com/tracker/PyDev/1174, so, please tackle that issue along with this one -- i.e.: create tests/fix it in this same pull request).